docs/application/dotnet/guides/applications/application_lifecycle.md

```diff
@@ -0,0 +1,186 @@
+
+- A new application is launched by the request of your application.
+- The user requests to go to the home screen.
+- A system event (such as an incoming phone call) occurs and causes a resident application with a higher priority to become active and temporarily hide your application.
+- An alarm is triggered for another application, which becomes the topmost window and hides your application.
+
+The application in the background goes into a suspended state. In the suspended state, the application process is executed with limited CPU resources. In other words, the platform does not allow the running of background applications, except for some exceptional applications (such as Media and Download) that necessarily work in the background. In this case, you can [define a background category in the application manifest file](./ui-app.md#allow_bg), to prevent the application from going into the suspended state.
+
+When your application becomes visible again, the `OnResume()` method is invoked. The visibility returns, when:
+
+- Another application requests your application to run (for example, the Task Navigator, which shows all running applications and lets the user select any application to run).
+- All applications on top of your application in the window stack finish.
+- An alarm is triggered for your application, bringing it to the front and hiding other applications.
+
+When your application starts exiting, the `OnTerminate()` method is invoked. Your application can start the termination process, when:
+
+- Your application itself requests to exit by calling the `Exit()` method to terminate the event loop.
+- The low memory killer is killing your application in a low memory situation.
+
+The Tizen .NET application can be in one of several application states.
+
+The [Tizen.Applications](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.html) namespace defines 5 states with corresponding state change methods. A state change method is triggered after each state change: whenever the application is created, starts running, or is paused, resumed, or terminated. The application must react to each state change appropriately.
+
+**Table: Application states**
+
+| State        | Description                              |
+|------------|----------------------------------------|
+| `READY`      | Application is launched.                 |
+| `CREATED`    | Application starts the main loop.        |
+| `RUNNING`    | Application is running and visible to the user. |
+| `PAUSED`     | Application is running but invisible to the user. |
+| `TERMINATED` | Application is terminated.               |
+
+The following figure illustrates the application state transitions.
+
+**Figure: Application state transitions**
+
+![Application state transitions](./media/application_lifecycle_diagram.png)
+
```

Comment:
@annie-samsung 
This guide page is in form of example - description. An example comes first and general description comes later.

In other pages, generally the feature is described first, and a specific example to the description comes later : Based on thought that description - example order is easier to understand the subject.

What do you think of the structure of this page? Do you think this example - description structure is acceptable or changing to description - example structure is required?

Reply:
Fixed.